js/ui/text_box/action_button_collection/button.js

```diff
@@ -1,14 +1,14 @@
 import $ from "../../../core/renderer";
 
 export default class ActionButton {
-    constructor(name, editor, { location = "after" }) {
+    constructor(name, editor, options) {
```

Comment:
```suggestion
    constructor(name, editor, options = { location = "after" }) {
```
Is default value unnecessary?

Reply:
The default value is unnecessary in this place. We need location value before the button creation to choose correct buttons container. We apply default location in _renderButtons